### ISSUE TITLE
Custom Column Gap Feature Request

### ISSUE THREAD

**Issue by Flipback8** [elementor/elementor#6628]
I would like to be able to set a custom Column Gap.
vs having to select Narrow, No Gap, Extender, Wide, Wider. It would be nice to have another option that reads Custom Gap and allow us to type in a numerical value to specify Pixel Spacing.

## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by bainternet** [comment]
@Flipback8 	

Currently, you can do that with "No Gap" and Custom CSS. but wi will look into adding it.

**Comment by Flipback8** [comment]
I set option to "No Gap" and the only way I found to add the custom Gap is by adding:

.elementor-inner-section .elementor-column-gap-no .elementor-element-populated {
    padding: 30px;
}
ideally I would like to add a class like .custom-gap and add my code. ex:
.custom-gap {
column-gap: 40px;
}
 It won't work for me.

**Comment by Flipback8** [comment]
Please watch short Screen Video:
[](https://www.useloom.com/share/8fdc64363fdd4485b1b1f537af97ae75)

**Comment by ariona** [comment]
@Flipback8 Elementor is not using CSS3 columns properties for building its columns layout, so the column-gap will not work with it. Use padding instead.

**Comment by KneePham** [comment]
I'm also interested in the ability to adding multiple custom gap. Maybe some sort of hook to modify the current list.

It might be easier for a user to change through the dropdown instead of needing to go into the advanced settings and then pasting custom css.

**Comment by black-eye** [comment]
+1

**Comment by arielk** [comment]
Fix by this PR: https://github.com/elementor/elementor/pull/12056

**Comment by DorShahar** [comment]
This issue has been resolved in Elementor v3.1.0
Feel free to update
Thanks!
